File: 第九次/9-6/Score.cpp
#include"Score.h"

//調皮一下，設定初值
Score::Score() {
	math = 100;
	name = "DiuDiu";
}
Score::Score(string newName, int newMath) {
	setName(newName);
	setMath(newMath);
}

string Score::getName() {
	return name;
}
void Score::setName(string newName) {
	name = newName;
}

int Score::getMath() {
	return math;
}
void Score::setMath(int newMath) {
	math = newMath;
}

//輸出函數
void Score::print() {
	cout << name << " " << math << endl;
}

File: Python/第9章/9-5/Score.py
class Score:
    def __init__(self, name, math):
        self.__name = name
        self.__math = math
    def setName(self, name):
        self.__name = name
    def setMath(self, math):
        self.__math = math
    def getName(self):
        return self.__name
    def getMath(self):
        return self.__math